testsuite/Makefile

```diff
@@ -15,7 +15,11 @@
 
 .NOTPARALLEL:
 
-BASEDIR := $(shell pwd)
+BASEDIR := "$(shell pwd)"
```

Comment:
These quotes make me a bit nervous: the variable is also sorrounded by quotes also below (in the definition of `BASEDIR_HOST` and maybe in other places). Won't that cause problems?

Reply:
In the previous version I had removed the quotes in the definition of  ```BASEDIR_HOST``` below but the diff was less clear should I remove them from the definition of ```BASEDIR_HOST``` or ```BASEDIR``` ?